dags/kpi_dashboard.py

```diff
@@ -30,18 +33,23 @@
         email=['telemetry-alerts@mozilla.com', 'jklukas@mozilla.com']
     )
 
-    smoot_usage_new_profiles_v2 = bigquery_etl_query(
-        task_id='smoot_usage_new_profiles_v2',
+    simpleprophet_forecasts_mobile = simpleprophet_forecast(
+        task_id="simpleprophet_forecasts_mobile",
+        datasource="mobile",
         project_id='moz-fx-data-shared-prod',
-        destination_table='smoot_usage_new_profiles_v2',
         dataset_id='telemetry_derived',
+        table_id='simpleprophet_forecasts_mobile_v1',
+        owner="jklukas@mozilla.com",
+        email=["telemetry-alerts@mozilla.com", "jklukas@mozilla.com"],
     )
 
-    smoot_usage_new_profiles_compressed_v2 = bigquery_etl_query(
-        task_id='smoot_usage_new_profiles_compressed_v2',
-        project_id='moz-fx-data-shared-prod',
-        destination_table='smoot_usage_new_profiles_compressed_v2',
-        dataset_id='telemetry_derived',
+    wait_for_firefox_nondesktop_exact_mau28 = ExternalTaskSensor(
+        task_id="wait_for_firefox_nondesktop_exact_mau28",
+        external_dag_id="bqetl_nondesktop",
+        external_task_id="telemetry__firefox_nondesktop_exact_mau28_raw__v1",
+        check_existence=True,
+        execution_delta=timedelta(hours=14, minutes=45),
```

Comment:
`bqetl_nondesktop` runs at `0 1 * * *`, so the `execution_delta` should be this.

Reply:
If we also add an ExternalTaskSensor for firefox_desktop_exact_mau28 and for the fxa forecast, then we no longer need to have the weird schedule for this DAG. It can also be defined to run at 01:00 UTC.

Reply:
Added the fxa forecast and changed the scheduling interval